lib/fs.js

```diff
@@ -520,7 +520,6 @@ function tryStatSync(fd, isUserFd) {
   } finally {
```

Comment:
Get ready to run benchmarks then :) as I remember that some of these changes (`finally` -> `catch`) were rejected in the past. I think it was on timers.